Libraries/libcar/Tests/test_Writer.cpp

```diff
@@ -98,3 +98,158 @@ TEST(Writer, TestWriter)
     EXPECT_EQ(rendition_count, 1);
 }
 
+
+TEST(Writer, TestWriter100)
```

Comment:
Both. The first test checks a write with both code paths (resize and not).

The second test had one extra change which I didn't end up pushing, but I might later, that called an added API in libbom that checked that the "free list" was exactly consumed and tracked the number of resize/memmoves ops via counters in the context.